daffodil-test/src/test/resources/org/apache/daffodil/section06/namespaces/namespaces.tdml

```diff
@@ -910,7 +910,7 @@
     <tdml:document><![CDATA[333]]></tdml:document>
     <tdml:errors>
       <tdml:error>Schema Definition Error</tdml:error>
-      <tdml:error>More than one definition for name: vagueEle</tdml:error>
+      <tdml:error>More than one definition for name: {}vagueElem</tdml:error>
```

Comment:
Similar question here, this feel incorrect since `{}` explicity means no-namespace, but no-namespace was not suggested as the namespace to use for this test. It's even possible that `vagueElement` doesn't even exist with no-namespace, it chould have two different namespaces, in which case {} is confusing since it implies no-namespace.

Maybe related, I noticed that `RootSpec.toString` always outputs curly braces. Maybe that's where the curly braces are coming from? Maybe it should only output curly braces if ns is defined? That would better match our "extended syntax".